app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/ToolbarViewModel.kt

```diff
@@ -51,7 +51,7 @@ class ToolbarViewModel(
     val events = _events.hide()
 
     @Deprecated(message = "Use ToolbarViewModel.events for new code")
-    val legacyEvents: LiveData<Consumable<Action>> = LiveDataReactiveStreams
+    val legacyEvents: LiveData<Consumable<Action>> = LiveDataReactiveStreams // TODO update test, remove this
```

Comment:
There appear to be no consumers: can we remove this?

If not, nit: can you elaborate in the comment on what "update test" means? Does that mean move it from `legacyEvents` to `events`?